src/YouTubeCatppuccinLatte.user.css

```diff
@@ -4187,36 +3662,20 @@
         background-size: 100% !important;
         position: relative !important;
     }
-
     #sponsorBlockPopupLogo {
         width: 40px !important;
         height: 40px !important;
         padding-left: 40px !important;
     }
-
     /*for overlays*/
     .sponsorSkipLogo {
         width: 18px !important;
         height: 18px !important;
         padding-left: 18px !important;
     }
-
     .sponsorBlockTooltip {
         background-color: var(--hover-background) !important;
     }
-
-    /*Fix like, dislike and share icons and text using wrong color*/
-    .yt-spec-button-shape-next__icon,
```

Comment:
Same as above, why was this fix removed?

Reply:
this was a temporary fix and it now appears correctly.